third_party/amd/python/triton_amd.cc

```diff
@@ -228,4 +229,17 @@ void init_triton_amd(py::module &&m) {
     }
     return false;
   });
+
+  m.def("has_matrix_core_feature", [](const std::string &arch) {
+    using mlir::triton::AMD::ISAFamily;
+    switch (mlir::triton::AMD::deduceISAFamily(arch)) {
+    case ISAFamily::CDNA3:
+    case ISAFamily::CDNA2:
+    case ISAFamily::CDNA1:
+    case ISAFamily::RDNA3:
```

Comment:
Do we need to further distinguish between mfma and wmma instructions?

Reply:
We can split when if the needs arise. For now this seems to cover the usage in the codebase.